generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>please help remove the "\" here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>It seems "\" is a special symbol in markdown, it's only used to change line, and will not show when open the notebook (not in this reivew NB)</p>

Reply:
Got it.